<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreatGraph</class>
 <widget class="QMainWindow" name="CreatGraph">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Graph</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pbt_dijkstra_alg">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>450</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Алгоритм Дейкстры</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbt_floyd_alg">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>450</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Алгоритм Флойда — Уоршелла</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbt_back">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>460</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Назад</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>521</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
